Gemfile

```diff
@@ -1,11 +1,39 @@
+require 'openssl'
 source 'https://rubygems.org'
+gemspec
 
-# gem 'activerecord', git: 'https://github.com/lego/ruby-on-rails.git'
+gem 'bcrypt' # used by the ActiveRecord test suite
 
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+if ENV['RAILS_SOURCE']
+  gemspec path: ENV['RAILS_SOURCE']
+else
+  def get_version_from_gemspec
+    require 'net/http'
+    require 'yaml'
+    spec = eval(File.read('activerecord-cockroachdb-adapter.gemspec'))
+    ver = spec.dependencies.detect{ |d|d.name == 'activerecord' }.requirement.requirements.first.last.version
```

Comment:
Can this be done more cleanly?

Reply:
Hmm...I'm not sure that it can but lemme give it a try.

Reply:
I did a little refactor in https://github.com/cockroachdb/activerecord-cockroachdb-adapter/pull/38/commits/130bff27bde546b4686e9875cf27110ed6b98c80 to make it a little clearer what's going. Getting the version from the gemspec is still a little hairy because that's what the Rubygems API gives us, but the rest of the file should be easier to follow 😄.